### ISSUE TITLE
failed to mount FUSE fs: exit status 1 when run in github runner on osx11

### ISSUE THREAD

**Issue by whitewhidow** [osxfuse/osxfuse#740]
hi,

When trying to perform a mount action using `rclone` on an **osx11(big-sur) github runner**, without installing osxfuse first, it will ofcourse error saying it needs osxfuse, which makes sense, however.

When i do in fact install osxfuse, it still errors saying:
`failed to mount FUSE fs: exit status 1`

_Now, i know that osxfuse seems to normally require a reboot, **however**,_

**i have the same thing successfully running on a catalina gthub runner**, and this works just fine (complains if osxfuse is missing, _yet mounts perfectly fine without reboot after installing osxfuse via brew_)

So i am wondering:
- is this error happening because it indeed needs a reboot on big-sur, which it seems to not need on catalina, or is this not at all related to needing a reboot?
- I have one user getting this same error, on catalina, even after a reboot, is this sure to be the same thing, or could this error mean several things
- Is there a place where i might find any usefull logs ? 

so in short to recap : i am doing the exact same thing on catalina github runner as on a big-sur gitlab runner, but it always crashes on the big-sur, and always passes on catalina, yet a very few select actual catalina users are also getting this error

i have gone trough atleast the 10 first pages of google results and tried everything i could think of so ANY help is very much appreciated !!!

thx

**Comment by KORraNpl** [comment]
@whitewhidow AFAIK, restart is needed on Big Sur, it's a new thing. See: https://developer.apple.com/documentation/macos-release-notes/macos-big-sur-11_0_1-beta-release-notes

> Installing a new kernel extension requires signing in as an Admin user. You must also restart your Mac to load the extension.

**Comment by bfleischer** [comment]
You need to follow the instructions displayed by the FUSE installer/mounter and macOS Big Sur. Big Sur will tell you if a reboot is required, when allowing the kernel extension to load in system preferences. Big Sur requires a restart before new or updated kernel extensions can be loaded.

**Comment by whitewhidow** [comment]
ah ok ,   so so far,   this is normal and expected behaviour then right? > (no reboot for catalina needed, bir-sur does need reboot)

outside of the normal install, is there ANY way to get this kernel extention loaded without a reboot AT ALL on big-sur (like modprobe on linux), that i could use to have my CI tests not fail (they cannot be rebooted, so will never be able to mount now ?))

thx for the responses so far !!!

**Comment by whitewhidow** [comment]
> > Installing a new kernel extension requires signing in as an Admin user. You must also restart your Mac to load the extension.

when using the actual gui,  that actuall seems to also be the case on catalina, ask you to unlock settings and allow unsigned things, and also tells me to reboot (even tho its not actually needed appearantly...)

, yet the gitlab catalina runner does all this, and works, without a reboot or manual interaction

**Comment by bfleischer** [comment]
Big Sur builds a cache containing all allowed kernel extensions. It will only load extensions from that cache. Swapping the cache requires a reboot. There is no obvious way around the reboot requirement. Please raise this issue with Apple.

**Comment by whitewhidow** [comment]
ok thx !

**Comment by whitewhidow** [comment]
sorry to bother you with one final question,   any idea on a timeline as to when the new (currently)prerelease version will appear in brew ? (are we talking days, or probably more like weeks atleast)

**Comment by bfleischer** [comment]
I'm usually promoting new releases to stable after one or two weeks, if no bugs are reported. I cannot speak for Homebrew.
